sanity/sanity-utils.ts

```diff
@@ -0,0 +1,17 @@
+import { createClient, groq } from "next-sanity";
+import { apiVersion, dataset, projectId } from './env'
+
+export async function getTeamMembers(): Promise<TeamMember[]> {
+    const client = createClient({
+        projectId,
+        dataset,
+        apiVersion
+    })
+    return client.fetch(groq`*[_type == "team-member"]{
```

Comment:
I have no idea how sanity works, maybe for other people some comments on what each parameter and what this does would be helpful :)